<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qtvirt_board</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="LedLayout">
     <item>
      <layout class="QVBoxLayout" name="ll1"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="ll2"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="ll3"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="ll4"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="ll5"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>160</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="SevSegLayout">
     <item>
      <widget class="QGraphicsView" name="sevseg1_view">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="sceneRect">
        <rectf>
         <x>0.000000000000000</x>
         <y>0.000000000000000</y>
         <width>0.000000000000000</width>
         <height>0.000000000000000</height>
        </rectf>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="sevseg2_view"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>chip.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="tml">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Время:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="time_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>0.0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="start_btn">
           <property name="text">
            <string>Старт</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stop_btn">
           <property name="text">
            <string>Стоп</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="BtnLayout">
     <item>
      <layout class="QVBoxLayout" name="lb1"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="lb2"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="lb3"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="lb4"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="lb5"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>691</width>
      <height>421</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>backgrnd.png</pixmap>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>horizontalLayoutWidget_4</zorder>
   <zorder>label</zorder>
   <zorder>frame</zorder>
   <zorder>horizontalLayoutWidget_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>694</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actModAuto"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="actSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actModAuto">
   <property name="text">
    <string>Открыть модуль...</string>
   </property>
  </action>
  <action name="actModMan">
   <property name="text">
    <string>Открыть модуль(Вручную)...</string>
   </property>
  </action>
  <action name="actSettings">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
